modules/ROOT/pages/custom-response-policy-example.adoc

```diff
@@ -222,17 +245,27 @@ Finally, we continue to complete the logic of our template:
 </choice>
 ----
 
-To modify the payload and/or the status code if they were set, we use the `http-policy-transform-extension that is included in the dependencies at the beginning of this section.
+To modify the payload and the status code, if set, you use the `http-policy-transform-extension` that is included in the dependencies at the beginning of this section. *<Lorena, where is this? Can we highlight>*
```

Comment:
I don't understand this question. 🤔

Reply:
I mean where exactly is this is section in the beginning. We are trying to be specific instead of sending them to go search for it. Makes sense?

Reply:
Oh, great. The section I'm refering to is the one titled `Add Dependencies in POM.XML File`

Reply:
Done! Thanks!